package-lock.json

```diff
@@ -31,8 +31,8 @@
       }
     },
     "aws-sdk": {
-      "version": "2.620.0",
-      "resolved": "https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.620.0.tgz",
+      "version": "2.991.0",
+      "resolved": "https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.991.0.tgz",
       "integrity": "sha512-j+PGKbYwk5OjzbCFbxzRkcJVEvfdn9YFkrU8dJdjleYf9AzUSUpTjMyZsyUPOKxJxdNdA9FYTs43WdtSKHZa0A==",
```

Comment:
how come the hash wasn't updated? Did you test this with `npm i`?

Reply:
Sorry @alexjurkiewicz, my bad; I was just lazy. It is updated, `npm` just installs it.